Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/nhinclib/NhincConstants.java

```diff
@@ -126,6 +126,12 @@ public static NHIN_SERVICE_NAMES fromValueString(String valueString) {
     public static final String AUTH_FRWK_NAME_ID_FORMAT_WINDOWS_NAME = "urn:oasis:names:tc:SAML:1.1:nameid-format:WindowsDomainQualifiedName";
     // SAML constants
     public static final String SAML_DEFAULT_ISSUER_NAME = "CN=SAML User,OU=SU,O=SAML User,L=Los Angeles,ST=CA,C=US";
```

Comment:
Remove the variable in OpenSAML2ComponentBuilder and start to use nhinconstant#SAML_DEFAULT_ISSUER_NAME